src/main/AndroidManifest.xml

```diff
@@ -138,7 +137,9 @@ limitations under the License.
 
         <activity android:name=".TrackEditActivity" />
 
-        <activity android:name=".TrackListActivity">
+        <activity
+            android:name=".TrackListActivity"
+            android:theme="@style/SplashTheme">
```

Comment:
I get the following error when I try to build:

AAPT: error: resource style/SplashTheme (aka de.dennisguse.opentracks.debug:style/SplashTheme) not found.